amundsen_application/static/js/components/Feedback/FeedbackForm/RatingFeedbackForm/index.tsx

```diff
@@ -48,14 +48,19 @@ export class RatingFeedbackForm extends AbstractFeedbackForm {
             </div>
           </div>
         </div>
-        <textarea
-          className="form-control form-group"
-          name="comment"
-          form={AbstractFeedbackForm.FORM_ID}
-          rows={8}
-          maxLength={2000}
-          placeholder={COMMENTS_PLACEHOLDER}
-        />
+        <div className="form-group">
+          <label htmlFor="comment">{COMMENTS_PLACEHOLDER}</label>
```

Comment:
We could give an FYI to design about this by leaving a comment on the old design that didn't include any label. Will try to find the link for you assuming it is in Figma.

Reply:
Yeah, I took the liberty of adding the label (I saw it wasn't on the design).

I get the impression (something we discussed some weeks ago I think) that designers think the placeholder attribute is enough for offering this information, but they are not aware the placeholder pattern doesn't really work for a11y.

I will add the comment, thanks!

Reply:
I found it in "Master -- Screens & UI Library", and it turns out recommendation in Figma does have a label. So we're set:
![image](https://user-images.githubusercontent.com/1790900/84736322-713d9480-af5a-11ea-97e2-6e32a50516fe.png)

Reply:
Awesome!